apps/cookbook/src/app/examples/calendar-example/examples/in-card.ts

```diff
@@ -16,11 +16,10 @@ const config = {
     [todayDate]="setTodayDate ? todayDate : null"
     [selectedDate]="selectedDate"
     (dateChange)="onDateChange($event)"
-    [yearNavigatorOptions]="showYearNavigator ? yearNavigatorOptions : null"
   ></kirby-calendar>
 </kirby-card>
 
-<kirby-card [hasPadding]="true">
+<kirby-card [hasPadding]="true" variant="outlined">
```

Comment:
😮

Reply:
Otherwise it looks too much like a part of the calendar in my opinion 😄 
<img width="354" alt="image" src="https://github.com/user-attachments/assets/1304c56a-fd14-4c9f-9451-d3bef030b543">